__tests__/jest-preset.spec.js

```diff
@@ -1,3 +1,16 @@
+/*
+ * Copyright (c) 2017 American Express Travel Related Services Company, Inc.
```

Comment:
2017? I don't know what's the rule about the years in the copyright

Reply:
Added this here because this was when the file was created, it was probably missed initialy